src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/RecordedPerformanceEntries.kt

```diff
@@ -6,5 +6,6 @@ package com.atlassian.performance.tools.jiraactions.api.w3c
  */
 class RecordedPerformanceEntries internal constructor(
     val navigations: List<PerformanceNavigationTiming>,
-    val resources: List<PerformanceResourceTiming>
+    val resources: List<PerformanceResourceTiming>,
+    val elements: List<PerformanceElementTiming>?
```

Comment:
So far, if navigations or resources were missing, they were an empty list.
Let's do the same for elements for consistency.
If we find that nullable lists are a better pattern, we should do it for all the fields in another PR.

Reply:
I did that because I thought it would not be a breaking change. Is it OK to just add a new field here from the API perspective?

Reply:
Yes, because the constructor is internal